package.json

```diff
@@ -81,7 +81,7 @@
   "scripts": {
     "build-node": "node ./scripts/build-node.js",
     "build:app:docker": "REACT_APP_DISABLE_SENTRY=true react-scripts build",
-    "build:app": "REACT_APP_GIT_SHA=$NOW_GITHUB_COMMIT_SHA react-scripts build",
+    "build:app": "REACT_APP_GIT_SHA=$VERCEL_GIT_COMMIT_SHA react-scripts build",
```

Comment:
I haven't used CRA in general.. why do we even need this? `REACT_APP_GIT_SHA`

Reply:
CRA requires env variables to be prefixed with `REACT_APP` if we want them exposed to the app during build.